### PR TITLE
BMW: Replace button entity with homeassistant service

### PR DESCRIPTION
**Pull request by rikroe**
## Proposed change

Adjust manual to deprecation/replacement of refresh button with `homeassistant.update_entity` service after migrating to `DataUpdateCoordinator`.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/67003
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by davet2001 — APPROVED** [review]
Looks good! Thanks @rikroe 👍

File: source/_integrations/bmw_connected_drive.markdown [hunk 1]
```diff
@@ -129,9 +129,18 @@ The `button.<your_vehicle>_find_vehicle` button requests the vehicle to update t
 
 </div>
 
-### Update the state / refresh from API
+## Update the state / refresh from API
 
-The `button.<vehicle_model>_refresh_from_api` button fetches the last state of the vehicles of all your accounts from the BMW server. This does *not* trigger an update from the vehicle; it gets the data from the BMW servers. So this service does *not* interact with your vehicles.
+To force a refresh from the BMW API to Home Assistant, execute the `homeassistant.update_entity` service on an arbitrary entity such as the lock or any sensor.
+
+
```
Comment (davet2001) [inline source/_integrations/bmw_connected_drive.markdown:135]: Unnecessary newline?

**Review by davet2001 — CHANGES_REQUESTED** [review]
Minor correction to formatting needed.

File: source/_integrations/bmw_connected_drive.markdown (hunk 1 above)
Comment (rikroe) [reply source/_integrations/bmw_connected_drive.markdown:135]: Done!

File: source/_integrations/bmw_connected_drive.markdown [hunk 2]
```diff
@@ -129,9 +129,17 @@ The `button.<your_vehicle>_find_vehicle` button requests the vehicle to update t
...
-The `button.<vehicle_model>_refresh_from_api` button fetches the last state of the vehicles of all your accounts from the BMW server. This does *not* trigger an update from the vehicle; it gets the data from the BMW servers. So this service does *not* interact with your vehicles.
+To force a refresh from the BMW API to Home Assistant, execute the `homeassistant.update_entity` service on an arbitrary entity such as the lock or any sensor.
+
+```yaml
+...
+service: homeassistant.update_entity
+target:
+  entity_id: binary_sensor.<your_vehicle>_windows
+```
+This does *not* trigger an update from the vehicle; it gets the data from the BMW servers, so this service does *not* interact with your vehicles.
```
Comment (klaasnicolaas) [inline source/_integrations/bmw_connected_drive.markdown:143]:
```suggestion

This does *not* trigger an update from the vehicle; it gets the data from the BMW servers, so this service does *not* interact with your vehicles.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/bmw_connected_drive.markdown (hunk 2 above)
Comment (rikroe) [reply source/_integrations/bmw_connected_drive.markdown:143]: Removed the sentence, but changed API to servers to make the information more accessible.

Comment (klaasnicolaas) [reply source/_integrations/bmw_connected_drive.markdown:143]: That was not wat I suggested, it's the extra blank line after the code block 😉

Comment (rikroe) [reply source/_integrations/bmw_connected_drive.markdown:143]: Aah oops 😅 seems I was to quick adjusting it from the phone 🙄

**Review by davet2001 — APPROVED** [review]
Looks good to me

**Review by klaasnicolaas — APPROVED** [review]
Let's merge it 🏅

File: source/_integrations/bmw_connected_drive.markdown [hunk 3]
```diff
@@ -129,9 +129,18 @@ The `button.<your_vehicle>_find_vehicle` button requests the vehicle to update t
 
 </div>
 
-### Update the state / refresh from API
+## Update the state / refresh from API
```
Comment (frenck) [inline source/_integrations/bmw_connected_drive.markdown:132]: This should not be documented on an integration basis. This is not functionality of this integration, but functionality provided by Home Assistant Core. If all integration would document this on their own, it would become quite a mess.

Please revert this change @rikroe


CC @klaasnicolaas @davet2001

Comment (davet2001) [reply source/_integrations/bmw_connected_drive.markdown:132]: Argh, sorry I did not realise that this was an issue.  

Would a better alternative be a more general comment to say that homeassistant.update_entity can be used to force a refresh if needed, without going into detail, and maybe linking to https://www.home-assistant.io/docs/scripts/service-calls/#homeassistant-services ?

Comment (frenck) [reply source/_integrations/bmw_connected_drive.markdown:132]: @davet2001 Yeah I think we should maybe introduce a page separately on the matter even?
